src/bin/pg_autoctl/cli_drop_node.c

```diff
@@ -578,7 +578,9 @@ cli_drop_local_node(KeeperConfig *config, bool dropAndDestroy)
 	 */
 	pid_t pid = 0;
 
-	if (!wait_for_pid_to_exit(config->pathnames.pid, 30, &pid))
+	int quit_timeout = dropForce ? 0 : 30;
```

Comment:
I think this requires some comments, I'm not sure what's your intention there, or why we need different timeouts depending on whether the user is using `--force` or not?